src/components/sbb-link-list/sbb-link-list.e2e.ts

```diff
@@ -1,13 +1,53 @@
-import { newE2EPage } from '@stencil/core/testing';
+import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
 
 describe('sbb-link-list', () => {
-  let element, page;
+  let element: E2EElement, page: E2EPage;
 
-  it('renders', async () => {
+  beforeEach(async () => {
     page = await newE2EPage();
-    await page.setContent('<sbb-link-list></sbb-link-list>');
-
+    await page.setContent(`
+      <sbb-link-list title-level="2">
+        <span slot="title">Help &amp; Contact</span>
+        <sbb-link href='https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html'>Rückerstattungen</sbb-link>
+        <sbb-link href='https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html'>Fundbüro</sbb-link>
+        <sbb-link href='https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html'>Beschwerden</sbb-link>
+        <sbb-link href='https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html'>Lob aussprechen</sbb-link>
+        <sbb-link href='https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html'>Sachbeschädigung melden</sbb-link>
+      </sbb-link-list>
+    `);
     element = await page.find('sbb-link-list');
+  });
+
+  it('renders', async () => {
     expect(element).toHaveClass('hydrated');
   });
+
+  describe('property sync', () => {
+    it('should sync properties/attributes with sbb-link', async () => {
+      await page.waitForChanges();
+      const links = await page.findAll('sbb-link-list sbb-link');
+      expect(
+        links.every(
+          (l) =>
+            l.getAttribute('text-size') === 's' &&
+            l.getAttribute('variant') === 'block' &&
+            l.getAttribute('negative') === null
+        )
+      ).toBeTruthy();
+    });
+
+    it('should update attributes with text-size m', async () => {
+      element.setAttribute('text-size', 'm');
+      await page.waitForChanges();
+      const links = await page.findAll('sbb-link-list sbb-link');
+      expect(links.every((l) => l.getAttribute('text-size') === 'm')).toBeTruthy();
+    });
+
+    it('should update attributes with negative', async () => {
+      element.setAttribute('negative', '');
```

Comment:
maybe it's clearer to set it to "true" ?

Reply:
No, I don't think so, as this should work like this.